packages/flutter/lib/src/material/tabs.dart

```diff
@@ -1089,7 +1098,8 @@ class TabBarView extends StatefulWidget {
   /// will be used.
   final TabController controller;
 
-  /// One widget per tab.
+  /// One widget per tab. Its length must match the length of the [TabBar.tabs]
```

Comment:
Insert a blank line after "One widget per tab". The format for docs should be:
```
/// Brief but surprisingly comprehensive summary statement (one sentence).
///
/// Additional explanation.
```

Reply:
Sounds good!